stan/math/prim/fun/identity_free.hpp

```diff
@@ -13,13 +13,32 @@ namespace math {
  * <p>This function is a no-op and mainly useful as a placeholder
  * in auto-generated code.
  *
- * @tparam T type of value
- * @param[in] y value
+ * @tparam Scalar type of value
```

Comment:
This is not a scalar any more, because now scalars include complex numbers.  Going back to `T` works.  Or if you insist on verbosely naming types, it should be `Real`.

Reply:
Why not? Complex scalars are scalars as well.

Reply:
The problem's the other way around.  These functions won't work for complex inputs, so the required type should be called `Real`, not `Scalar`.